src/components/SocialShare/SocialShare.js

```diff
@@ -9,7 +10,7 @@ function SocialShare(props) {
                 target="_blank"
                 rel="noopener noreferrer"
             >
-                <i className="fab fa-facebook"></i> Facebook
+                <i className="fab fa-facebook-f"></i>
```

Comment:
typo?

Reply:
No. The Facebook-f icon comes without the blue box around it. Made it fit better with the other icons.